test_integration/test_frequency_hint_usage.py

```diff
@@ -50,10 +50,11 @@
 import geopmpy.error
 
 from test_integration import util
+from test_integration import geopm_test_launcher
```

Comment:
Why?

If this is wrong, it's wrong in the template too.

Reply:
The problem is that the test fixtures use the launcher in this test to get the min and max frequencies.